src/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
```

Comment:
I would remove everything from line 5 down. We're gonna simplify the JSX to just a simple h1 tag with App in it